packages/turbo-gen/src/templates/simple-ts/config.ts

```diff
@@ -8,19 +8,31 @@ export default function generator(plop: PlopTypes.NodePlopAPI): void {
       {
         type: "input",
         name: "file",
-        message: "What is the name of the file to create?",
-      },
-      {
-        type: "input",
-        name: "author",
-        message: "What is your name? (Will be added as the file author)",
+        message: "What is the name of the new file to create?",
+        validate: (input: string) => {
+          if (input.includes(".")) {
+            return "file name cannot include an extension";
+          }
+          if (input.includes(" ")) {
+            return "file name cannot include spaces";
+          }
+          if (!input) {
+            return "file name is required";
+          }
+          return true;
```

Comment:
```suggestion
          // allows alphanumeric chars, underscores, hyphens, and plus signs
          return /^[\w\-+]+$/test(input)
```
and:
- add any other special chars you might want to allow
- hoist that regex out of the function so it's not recreated on every call

Reply:
I actually broke this out intentionally so the error message that the user sees could be more specific instead of just "must match this regex". The performance different isn't noticeable, and I think the more specific error messages are worth the verbosity.